src/kakarot/instructions/stop_and_math_operations.cairo

```diff
@@ -250,14 +251,15 @@ namespace StopAndMathOperations {
         jmp end;
 
         SIGNEXTEND:
+        let bitwise_ptr = cast([fp - 4], BitwiseBuiltin*);
```

Comment:
undo this since you're not using the bitwise in the signextend anymore

Reply:
thx nice catch